outlines/exploring-graphiql-the-in-browser-ide-for-graphql.md

```diff
@@ -0,0 +1,87 @@
+# Exploring GraphiQL: The In-Browser IDE for GraphQL
+
+**Target Audience:** This article is for all developers and testers who integrate or test GraphQL APIs, whether front-end developers, back-end developers, or QA testers. It's also for anyone else interested in the world of GraphQL APIs.
+
+**Aim:** This article aims to provide developers with an in-depth understanding of GraphiQL, its features, and how to effectively use it to interact with GraphQL APIs. It will cover the fundamentals, and practical applications for utilizing GraphiQL as an in-browser IDE for GraphQL.
+
+## Introduction
+Why waste time and effort troubleshooting your GraphQL API directly in your application when there's a more efficient way? Imagine testing and perfecting your queries outside your app, ensuring they work flawlessly before implementation. Sounds ideal, right? With [GraphiQL](https://github.com/graphql/graphiql), you're saving time and gaining control over your development process, empowering you to work more confidently and effectively.
+
+## Prerequisites
+To follow along with this guide, you need:
+* Fundamental understanding of GraphQL 
+* Basic knowledge of schemas, queries, mutations, and resolvers. 
+* A web browser
+* A GraphQL API endpoint: This is how GraphiQL can communicate with a server to access the data. 
+
+## What is GraphiQL?
+* Definition and overview of GraphiQL
+
+## Why use GraphiQL?
+* Advantages/benefits of using GraphiQL (things like Improving your request, validating your improvements, and easily debugging requests running into problems)
+
+## Methods for accessing GraphiQL
+To access GraphiQL, you are not limited to the default in-browser method. 
+* Head to [https://tailcall.run/playground/?u=http://127.0.0.1:8000/graphql](https://tailcall.run/playground/?u=http://127.0.0.1:8000/graphql) by querying your search engine (browser) to access the GraphiQL online editor, which works with the Star Wars data, and it is publicly available for you to use
+* If you are using Gatsby, you can start the Gatsby development server by referencing this: (https://tailcall.run/docs/#starting-the-graphql-server)[https://tailcall.run/docs/#starting-the-graphql-server]
```

Comment:
> start the Gatsby development server

Doesn't make sense

Reply:
Alright. But is the first point okay?

Reply:
> > start the Gatsby development server
> 
> Doesn't make sense

Alright. I'd just remove that and add the part of starting a server from the link you shared earlier.

Reply:
which works with the Star Wars data, and it is publicly available for you to use is incorrect

Reply:
I'd just remove that part. The initial link supports the Star Wars data.

Reply:
> > which works with the Star Wars data, and it is publicly available for you to use
> 
> is incorrect

To clarify is there more than one method to access GraphiQL when using Tailcall?

Reply:
when you start the Tailcall server you get a link to playground.
or you can directly go to the https://tailcall.run/playground/ and give your graphql server url in input
Only thing you need to make sure your graphql server should allow cors for Tailcall domain

Reply:
> when you start the Tailcall server you get a link to playground. or you can directly go to the https://tailcall.run/playground/ and give your graphql server url in input Only thing you need to make sure your graphql server should allow cors for Tailcall domain

Alright, now I get it. Thank you for clarifying. I have committed the changes.